ui/client/components/graph/EspNode/getIconHref.ts

```diff
@@ -0,0 +1,9 @@
+import ProcessUtils from "../../../common/ProcessUtils"
```

Comment:
Maybe it should be named as icons.ts?

Reply:
I prefer to name modules by main exported member. as long as module contains only one function I'd leave this name. "icons" says nothing about content.

Reply:
`rect.ts` has also only one function.. So I would like to ask you for do it in one standard.